test-utils/src/lib.rs

```diff
@@ -243,5 +255,30 @@ fn to_sync_ciruit_witness<const SYNC_COMMITTEE_SIZE: usize>(
         .iter()
         .map(|b| b.deref().to_vec())
         .collect();
+
+    // Proof length is 3
+    let (attested_header_multiproof, attested_header_helper_indices) =
+        beacon_header_multiproof_and_helper_indices(
+            &mut args.attested_header.clone(),
+            &[Minimal::HEADER_SLOT_INDEX, Minimal::HEADER_STATE_ROOT_INDEX],
+        );
+    // Proof length is 4
+    let (finalized_header_multiproof, finalized_header_helper_indices) =
+        beacon_header_multiproof_and_helper_indices(
+            &mut args.finalized_header.clone(),
+            &[Minimal::HEADER_SLOT_INDEX, Minimal::HEADER_BODY_ROOT_INDEX],
+        );
+
+    args.finalized_header_multiproof = finalized_header_multiproof
+        .into_iter()
+        .map(|n| n.to_vec())
```

Comment:
for real, let's make it return Vec<Vec<u8>>
all this code repetition hurts 😖

Reply:
i like the pain